troposphere/secretsmanager.py

```diff
@@ -46,7 +46,7 @@ class RotationSchedule(AWSObject):
     props = {
         'SecretId': (basestring, True),
         'RotationLambdaARN': (basestring, True),
-        'RotationRules': ([RotationRule], False)
+        'RotationRules': (RotationRule, False)
```

Comment:
Looking at the docs and the resource spec files, the RotationRule property should be names RotationRules (not the plural). Could you make this change as well?